apps/system/style/notifications/notifications.css

```diff
@@ -146,15 +147,20 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
```

Comment:
Since we're keeping the `width:auto` I think we should switch the padding to `padding: 0 1.5rem` which means that there will be 1.5rem of padding on each side horizontally.